src/questions/owner_borrow.yaml

```diff
@@ -240,3 +240,751 @@
+      key: D
+
+- uuid: 17324731-55d5-4026-b268-0b79031fe113
+  type: SingleChoice
+  content: Ownership rules, There can only be ____ owner/s at a time
+  hint: NoHint
+  explanation: "Ownership belongs to an entity"
+  answerKeys:
+    - c
+  subTopics:
+    - ownership
+  difficultyLevel: low
+  choices:
+    - content: "0"
+      key: A
+    - content: "2"
+      key: B
+    - content: "1"
+      key: c
+    - content: "3"
+      key: D
+
+- uuid: a86625dd-2eba-4808-87c0-30cde8cf7c2d
+  type: SingleChoice
+  content: |
+    Complete the sentence
+    When the owner goes out of _____, the value will be ______.
+  hint: NoHint
+  explanation: "Owners have right and if they abandon the right it get dropped"
+  answerKeys:
+    - A
+  subTopics:
+    - ownership
+  difficultyLevel: low
+  choices:
+    - content: "scope, dropped"
+      key: A
+    - content: "dropped, scope"
+      key: B
+    - content: "revert, opened"
+      key: c
+    - content: "scope, opened"
+      key: D
+
+- uuid: e539b827-cbf2-4b35-8fcc-af16c8d0c375
+  type: SingleChoice
+  content: which statement best desribe reference?
+  hint: pointer
+  explanation: "point to valid value"
+  answerKeys:
+    - B
+  subTopics:
+    - borrowing
+  difficultyLevel: low
+  choices:
+    - content: "a set of rules that governs how a Rust program manages memory"
+      key: A
+    - content: "a reference is guaranteed to point to a valid value of a particular type for the life of that reference."
+      key: B
+    - content: "new concept for many programmers, it does take some time to get used to"
+      key: c
+    - content: "the range within a program for which an item is valid"
+      key: D
+
+- uuid: 3661c275-8559-40e4-b5dd-ac138e3c5be9
+  type: SingleChoice
+  content: The action of creating a reference to a variable is called ____
+  hint: NoHint
+  explanation: "creation of reference is pointer to a variable"
+  answerKeys:
+    - D
+  subTopics:
+    - borrowing
+  difficultyLevel: low
+  choices:
+    - content: "Referencing"
+      key: A
+    - content: "Ownership"
+      key: B
+    - content: "Heaping"
+      key: c
+    - content: "Borrowing"
+      key: D
+
+- uuid: 434bc0ff-02a9-4287-a469-e74c3b673f70
+  type: SingleChoice
+  content: What is a Scope?
+  hint: noHint
+  explanation: ""
+  answerKeys:
+    - A
+  subTopics:
+    - memory-allocation
+  difficultyLevel: low
+  choices:
+    - content: "the range within a program for which an item is valid"
+      key: A
+    - content: "the range within a program for which an item is invalid"
+      key: B
+    - content: "the range within a program"
+      key: c
+    - content: "None of the above"
+      key: D
+
+- uuid: 896206c4-09f5-4c46-8cfd-6f5597df0c02
+  type: SingleChoice
+  content: When your code calls a function, the values passed into the function are pushed into
+  hint: not a first in first out
+  explanation: "function are pushed in and dont have to leave first"
```

Comment:
`function are pushed in and dont have to leave first`. this doesn't explain the question